todo_app/data/mongo_items.py

```diff
@@ -0,0 +1,51 @@
+import os
+import dotenv
+from bson import ObjectId
+import pymongo
+
+from todo_app.data.Item import Item
+
+dotenv.load_dotenv()
```

Comment:
This line makes me slightly nervous - it's a definite side effect in this file, and it would be easy for it to lead to tests accidentally loading real environment variables and doing things they shouldn't!

When I remove this the app seems to still work without issue, so I _think_ we can just remove this?

Reply:
I have removed it